CovertInfiltration/Src/CovertInfiltration/Classes/UICovertActionsGeoscape.uc

```diff
@@ -750,11 +760,16 @@ simulated function UpdateCovertActionInfo()
 
 simulated protected function UpdateSlots()
 {
+	local XComGameState_CovertAction CurrentAction;
 	local UICovertActionsGeoscape_SlotInfo SlotInfo;
 	local UICovertActionsGeoscape_SlotRow Row;
 	local UICovertActionsGeoscape_Slot SlotUI;
 	local int iCurrentSlot, iCurrentRow;
 	local int TotalNeededRows;
+		
+	CurrentAction = GetAction();
+
+	InfiltrationMissionLabel.SetVisible(class'X2Helper_Infiltration'.static.IsInfiltrationAction(CurrentAction) && !CurrentAction.bStarted);
```

Comment:
You need to hide the list as well:
```uc
ActionSlotRows.SetVisible(!InfiltrationMissionLabel.bVisible);
```
Otherwise it will be a mess (since inflil actions have 6 slots - 3 rows)